src/mongo/unittest/unittest.h

```diff
@@ -368,6 +411,42 @@ class Test {
     logger::MessageLogDomain::AppenderAutoPtr _captureAppender;
 };
 
+class Benchmark : public Test {
+protected:
+    int trial_num = 0;
+
+public:
+    /**
+     * Called to prepare the benchmark for execution
+     */
+    virtual void run();
```

Comment:
I'd declare this like virtual Stats run() final;

We don't want fixtures to override Benchmark::run(), even though they are allowed to override Test::run(). Also by returning Stats, we don't need to add it to a global list which is generally in poor taste.